tree/30_generic_methods/file_from_template_jinja2.cf

```diff
@@ -161,23 +161,23 @@
 # Use the following template:
 #
 # ```jinja2
-# {% for item in vars.variable_prefix.dict_name.hosts %}
+# {% for item in vars.my_prefix.my_name.hosts %}
```

Comment:
why renaming the variable?

Reply:
My bad, I had at first a missleading error, and though that the name "variable_prefix" was already in use. I double checked and there is no need to rename the variables.